website/docs/d/catalog_media.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_catalog_media"
+sidebar_current: "docs-vcd-data-source-catalog-media"
+description: |-
+  Provides a catalog media data source.
+---
+
+# vcd\_catalog\_media
+
+Provides a vCloud Director Catalog media data source. A Catalog media can be used to reference a catalog media and use its 
+data within other resources or data sources.
+
+Supported in provider *v2.5+*
+
+## Example Usage
+
+```hcl
+data "vcd_catalog_media" "existing-media" {
+  org     = "my-org"
+  catalog = "my-cat"
+  name    = "my-media"
+}
+
+output "media_size" {
+  value = data.vcd_catalog_media.size
```

Comment:
Is this a valid data access? I think it is missing resource name `existing-media`. Also I do not think we need these `output` fields in examples. People can access them directly using `data.vcd_catalog_media.existing-media.size`

Reply:
updated